quipucords/api/status/view.py

```diff
@@ -33,3 +40,12 @@ def status(request):
             env_dict[key] = value
     server_info["environment_vars"] = env_dict
     return Response(server_info)
+
+
+@api_view(["GET"])
+@csrf_protect
```

Comment:
w8, why this view requires csrf protection?

Reply:
overkill, not needed for a GET endpoint. just removed it